src/os/bluestore/BlueFS.cc

```diff
@@ -3176,8 +3176,8 @@ void BlueFS::_maybe_compact_log(std::unique_lock<ceph::mutex>& l)
 }
 
 int BlueFS::open_for_write(
-  const string& dirname,
-  const string& filename,
+  string_view dirname,
```

Comment:
suggest to use (or do not use) std:: prefix uniformly everywhere through the patch

Reply:
sure. now use `std::string_view` uniformly.